lib/data_update_scripts/20220815102733_backfill_community_emoji.rb

```diff
@@ -0,0 +1,10 @@
+module DataUpdateScripts
+  class BackfillCommunityEmoji
+    def run
+      return if Settings::Community.community_emoji.blank?
+
+      new_community_name = "#{Settings::Community.community_name} #{Settings::Community.community_emoji}"
```

Comment:
**Question:** Should we consider the scenario of the DUS executing on a Forem instance that already has the emoji in their name? I'm not sure if this is even necessary (if it's never meant to happen), but a quick guard clause here might make this DUS slightly less error prone if something like this were to happen

```ruby
return if Settings::Community.community_name.end_with(Settings::Community.community_emoji)
```

Reply:
I don't think it will hurt to do something like that, however we're also following up with a changelog post to let Forem owners know about the change and allow them to remove the community emoji form the name if they would like to. 

I was going to initially use `ends_with?` but then I decided to go with `include?` so that it will pick up if the emoji is anywhere in the string. If the emoji is in the community_name already, then I doubt they'll want us to add another one at the end. I added a test as well.